app/src/main/java/ru/alexzdns/fundamentals/homework/ui/fragments/MovieDetailsFragment.kt

```diff
@@ -0,0 +1,103 @@
+package ru.alexzdns.fundamentals.homework.ui.fragments
+
+import android.content.Context
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.RatingBar
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+import ru.alexzdns.fundamentals.homework.R
+import ru.alexzdns.fundamentals.homework.domain.ActorsDataSource
+import ru.alexzdns.fundamentals.homework.domain.MovieDataSource
+import ru.alexzdns.fundamentals.homework.ui.adapters.ActorsAdapter
+
+class MovieDetailsFragment : androidx.fragment.app.Fragment() {
+    private var listenerMovieDetails: MovieDetailsClickListener? = null
+
+    override fun onAttach(context: Context) {
+        super.onAttach(context)
+        listenerMovieDetails = context as? MovieDetailsClickListener
+    }
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        val view = inflater.inflate(R.layout.fragment_movies_details, container, false)
+
+        if (arguments != null) {
+            val moviePosition = arguments?.getInt(MOVIE_ID)
+
+            if (moviePosition != null) {
+                val movie = MovieDataSource.getMovie()[moviePosition]
+
+                val title = view.findViewById<TextView>(R.id.mdf_tv_movie_title)
+                title.text = movie.title
```

Comment:
Локальные переменные тут излишни, можно обойтись без них. 
```suggestion
                view.findViewById<TextView>(R.id.mdf_tv_movie_title).text = movie.title
```